samples/net/download/prj.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+
+CONFIG_DOWNLOAD_CLIENT=y
+CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
+
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Is there a reason to not use the default C library?

Reply:
You should remove this line here and override the default where you need to.
If I remove this line and the override in the nRF9 series boards, then I get picolibc by default (as intended).